velox/vector/FlatVector-inl.h

```diff
@@ -100,7 +100,7 @@ std::unique_ptr<SimpleVector<uint64_t>> FlatVector<T>::hashAll() const {
 
   // overwrite the null hash values
   if (!BaseVector::nullCount_.has_value() ||
-      BaseVector::nullCount_.value() > 0) {
+      BaseVector::getNullCount().value() > 0) {
```

Comment:
should we maybe simplify this as:

```
if (getNullCount().value_or(0) > 0) {
```

Reply:
Good idea, I have refractor it.

Old logic is `(NOT has_value) OR (value > 0)`, so we use `value_or(1) > 0` instead.